doc/sphinx/Pacemaker_Explained/status.rst

```diff
@@ -1,375 +1,374 @@
-   
-   | in_ccm |            
-   indexterm:[in_ccm,Node Status]
-   indexterm:[Node,Status,in_ccm]
-   Is the node a member at the cluster communication layer? Allowed values:
-   +true+, +false+.
-   
-   | crmd |
-   indexterm:[crmd,Node Status]
-   indexterm:[Node,Status,crmd]
-   Is the node a member at the pacemaker layer? Allowed values: +online+,
-   +offline+.
-   
-   | crm-debug-origin |
-   indexterm:[crm-debug-origin,Node Status]
-   indexterm:[Node,Status,crm-debug-origin]
-   The name of the source function that made the most recent change (for debugging
-   purposes).
-   
-   | join |
-   indexterm:[join,Node Status]
-   indexterm:[Node,Status,join]
-   Does the node participate in hosting resources? Allowed values: +down+,
-   +pending+, +member+, +banned+.
-               
-   | expected |
-   indexterm:[expected,Node Status]
-   indexterm:[Node,Status,expected]
-   Expected value for +join+.
-   
-   |=========================================================
-   
-   The cluster uses these fields to determine whether, at the node level, the
-   node is healthy or is in a failed state and needs to be fenced.
-   
-   == Transient Node Attributes ==
-   
-   Like regular <<s-node-attributes,node attributes>>, the name/value
-   pairs listed in the +transient_attributes+ section help to describe the
-   node.  However they are forgotten by the cluster when the node goes offline.
-   This can be useful, for instance, when you want a node to be in standby mode
-   (not able to run resources) just until the next reboot.
+.. table:: **Authoritative Sources for State Information**
+
+   +----------------------+----------------------+
+   | CIB Object           | Authoritative Source |
+   +======================+======================+
+   | node_state           | pacemaker-controld   |
+   +----------------------+----------------------+
+   | transient_attributes | pacemaker-attrd      |
+   +----------------------+----------------------+
+   | lrm                  | pacemaker-execd      |
+   +----------------------+----------------------+
+
+The fields used in the ``node_state`` objects are named as they are
+largely for historical reasons and are rooted in Pacemaker's origins
+as the resource manager for the older Heartbeat project. They have remained
+unchanged to preserve compatibility with older versions.
          
-   In addition to any values the administrator sets, the cluster will
-   also store information about failed resources here.
+.. table:: **Node Status Fields**
+
+   +------------------+----------------------------------------------------------+
+   | Field            | Description                                              |
+   +==================+==========================================================+
+   | id               | .. index:                                                |
+   |                  |    single: id; Node Status                               |
```

Comment:
I'm thinking lowercase for all generic index terms, to help distinguish them from technical terms where capitalization is significant (like PCMK_debug, IPAddr2, etc.)